tox & runtest.py leading to bad marshal data ValueError

If I run `tox` — to test everything — followed by `runtests.py` — to test what I'm working on more quickly — I get a `ValueError` (or lots of them) as `pytest` tries to collect tests. 

```
../../../.pyenv/versions/drf2.7/lib/python2.7/site-packages/py/_path/local.py:642: in pyimport
    __import__(modname)
../../../.pyenv/versions/drf2.7/lib/python2.7/site-packages/_pytest/assertion/rewrite.py:134: in find_module
    co = _read_pyc(fn_pypath, pyc)
../../../.pyenv/versions/drf2.7/lib/python2.7/site-packages/_pytest/assertion/rewrite.py:311: in _read_pyc
    co = marshal.load(fp)
E   ValueError: bad marshal data (unknown type code)
```

Noting the `in _read_pyc` I resolve the issue simply: 

```
 $ find . -name '*.pyc' -delete
```

I'm just wondering if this would be handy as part of `runtests.py`? (Anyone else bumping into this?)

If so I'm happy to make the pull request. If not do close.

--- a/tox.ini
+++ b/tox.ini
@@ -9,6 +9,8 @@ envlist =
 
 [testenv]
 commands = ./runtests.py --fast
+setenv =
+       PYTHONDONTWRITEBYTECODE=1
 
 [testenv:flake8]
 basepython = python2.7

